From 13e3fd3cd08146ac573714f318c354d497683100 Mon Sep 17 00:00:00 2001
From: Arun Suresh <asuresh@cloudera.com>
Date: Thu, 30 Oct 2014 13:46:14 -0700
Subject: [PATCH 0025/1164] CLOUDERA_BUILD: CDH-22860 - Update HIVE metastore
 startup scripts to add sentry plugin to classpath

(cherry picked from commit df7d140c2a3bc39807c510e50b010fcf90b615fd)
---
 bin/ext/metastore.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/ext/metastore.sh b/bin/ext/metastore.sh
index b294942..abeaad4 100644
--- a/bin/ext/metastore.sh
+++ b/bin/ext/metastore.sh
@@ -34,7 +34,7 @@ metastore() {
   fi
   if [[ -n "$SENTRY_HOME" ]]
   then
-    for f in ${SENTRY_HOME}/lib/*.jar; do
+    for f in ${SENTRY_HOME}/lib/*.jar ${SENTRY_HOME}/lib/plugins/*.jar; do
       export HADOOP_CLASSPATH=${HADOOP_CLASSPATH}:${f}
     done
   fi
-- 
1.7.9.5

